app/models/course/level.rb

```diff
@@ -20,6 +20,14 @@ def default_level?
     experience_points_threshold == 0
   end
 
+  # Returns the next higher level in the course
+  # nil is returned if current level is the highest level
+  #
+  # @return [Course::Level, nil]
```

Comment:
@fonglh good catch! Okay let's stick with multiple return tags then. My documentation is a mismash of yardoc, javadoc, doxygen...